libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -105,6 +105,9 @@
 
     private static final String USER_EVENT_KEY_TEMPLATE = "template";
 
+    private static final String FILE_TYPE_NOT_PERMITTED_ERROR =
```

Comment:
Where does this error message  "_Sorry, this file type is not permitted for security reasons._" come from? I noticed below `onMediaUploadFailed`  method is expecting a `errorMessage` to have this value, but it wasnt clear to me here or the other PRs where that String sentence comes from. If it is server code or elsewhere in Gutenberg/WPAndroid, perhaps we should be using an error ID or something? Not a blocker but seems strange to matching against this long String. Maybe at least a comment explaining its origin would be helpful.

Reply:
Ah I see in your other comment now https://github.com/wordpress-mobile/WordPress-Android/pull/13635/files#r552846333